### PR TITLE
Ui fixes done in filter task page

### PR DESCRIPTION
**Pull request by Raisa-12**
### Functionality:
Make the UI fixes of filter in dark mode.. Fix there borders, position of custom filter

### Solution:
made changes in filter-tasks.component.css
made changes in filter-tasks.component.html
### Risk level:
- [ ] high 
- [X] medium
- [ ] low

### How to test:
ng serve
firebase emulators:start

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -136,5 +136,8 @@
       }
     }
   },
-  "defaultProject": "worktrolly-ui"
+  "defaultProject": "worktrolly-ui",
+  "cli": {
+    "analytics": false
```
Comment (Pendyala-thanvi) [inline angular.json:141]: remove these changes

**Review by Pendyala-thanvi — CHANGES_REQUESTED** [review]

**Review by viv9k — APPROVED** [review]
Approved

**Comment by Raisa-12** [comment]
Sure sir

**Comment by Pendyala-thanvi** [comment]
@Raisa-12 resolve merge conflicts

**Pull request merged**